Makefile

```diff
@@ -15,7 +15,7 @@ all: manager
 
 # Run tests
 test: generate fmt vet manifests
-	go test ./... -coverprofile cover.out
+	go test -v ./test/e2e -ginkgo.v
```

Comment:
Can we keep two separate makefile rules, one for e2e tests and one for unit tests?
I expect this to belong to the latter

Reply:
This go test ./... runs all the tests in the code base (generally, it's intended for unit tests)
I'd like to have a specific makefile rule that runs e2e tests and one for unit tests.

You can use build flags to avoid e2e tests to be caught by go test ./... 
See for example https://mickey.dev/posts/go-build-tags-testing/